examples/text-classification/run_glue.py

```diff
@@ -230,6 +230,10 @@ class ModelArguments:
         default=False,
         metadata={"help": "Will enable to load a pretrained model whose head dimensions are different."},
     )
+    add_pad_token: bool = field(
+        default=True,
```

Comment:
This way the pad_token is automatically applied to all the other models, and I'm not sure it might be acceptable. I would suggest to add it as 
`action="store_true",`
which means that you can add it or not, to the model. Please, adjust the subsequent code to reflect this change.

Reply:
While it is certainly saving time not adding more options to existing command line, as an user I would prefer to have it clear what I'm running. However, I don't have a strong opinion on it. To me, you can PR directly in the main, just have in mind this https://github.com/huggingface/optimum/blob/main/CONTRIBUTING.md instruction before opening the PR